src/main/java/com/selfxdsd/selfweb/api/output/JsonTask.java

```diff
@@ -42,10 +42,12 @@ public JsonTask(final Task task) {
         super(
             Json.createObjectBuilder()
                 .add("issueId", task.issueId())
+                .add("invoiceNumber", task.toString().split(":")[1])
```

Comment:
@amihaiemil Not a big fan of this... Maybe we could pass a `StatusTask` to `JsonTask` instead,
which in return would have an `invoiceId()` and  a `status()` method.